### PR TITLE
Add nightly 8.16 DRA builds

### PR DESCRIPTION
**Pull request by seanstory**
We weren't building DRA artifacts for 8.16 nightly

### CODE REVIEW

File: catalog-info.yaml [hunk 1]
```diff
@@ -57,6 +57,10 @@ spec:
           branch: 8.x
           cronline: '@daily'
           message: "Builds, tests, and pushes daily `8.x` DRA artifacts"
+        Daily 8.16:
+          branch: 8.x
```
[the remark this answers is not in the document]
Comment (seanstory) [reply catalog-info.yaml:61]:
```suggestion
          branch: 8.16
```
lol yep

**Comment by tpgilmore** [comment]
@seanstory this appears to be breaking Terrazzo since it's being interpreted as a float. The 8.16 should be wrapped in quotes, ref - https://buildkite.com/elastic/terrazzo/builds/8847#01932242-99ea-4d29-9767-d9877cfefff1

```
Traceback (most recent call last):
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/main.py", line 214, in <module>
    TerrazzoStack(
  File "/home/terrazzo/venv/lib/python3.11/site-packages/jsii/_runtime.py", line 118, in __call__
    inst = super(JSIIMeta, cast(JSIIMeta, cls)).__call__(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/terrazzo/constructs/stack.py", line 223, in __init__
    BuildkitePipelines(
  File "/home/terrazzo/venv/lib/python3.11/site-packages/jsii/_runtime.py", line 118, in __call__
    inst = super(JSIIMeta, cast(JSIIMeta, cls)).__call__(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/terrazzo/constructs/buildkite/pipelines.py", line 130, in __init__
    self.pipeline_schedule_resources = self.declare_pipeline_schedules()
                                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/terrazzo/constructs/buildkite/pipelines.py", line 554, in declare_pipeline_schedules
    pipeline_schedule_resources[schedule_id] = PipelineSchedule(
                                               ^^^^^^^^^^^^^^^^^
  File "/home/terrazzo/venv/lib/python3.11/site-packages/jsii/_runtime.py", line 118, in __call__
    inst = super(JSIIMeta, cast(JSIIMeta, cls)).__call__(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/imports/buildkite/pipeline_schedule/__init__.py", line 96, in __init__
    config = PipelineScheduleConfig(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/imports/buildkite/pipeline_schedule/__init__.py", line 382, in __init__
    check_type(argname="argument branch", value=branch, expected_type=type_hints["branch"])
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/imports/buildkite/pipeline_schedule/__init__.py", line 25, in check_type
    return typeguard.check_type(argname=argname, value=value, expected_type=expected_type) # type:ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/terrazzo/venv/lib/python3.11/site-packages/typeguard/__init__.py", line 785, in check_type
    raise TypeError(
TypeError: type of argument branch must be str; got float instead
Traceback (most recent call last):
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/.buildkite/synth.py", line 124, in <module>
    run("python main.py")
  File "/buildkite/builds/bk-agent-prod-k8s-1731446665611408489/elastic/terrazzo/.buildkite/synth.py", line 34, in run
    check_call(command.split(), stdin=stdin, stdout=stdout, stderr=stderr)
  File "/usr/lib/python3.11/subprocess.py", line 413, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['python', 'main.py']' returned non-zero exit status 1.
```

**Comment by tpgilmore** [comment]
More context to the above - https://elastic.slack.com/archives/C06V49U5HJB/p1731447108215989

**Comment by seanstory** [comment]
woops! fix: https://github.com/elastic/connectors/pull/2957

**Pull request merged**